libr/core/cmd_print.c

```diff
@@ -5852,6 +5852,93 @@ l = use_blocksize;
 				}
 			}
 			break;
+		case 't': // "pst"
+			if (l > 0) {
+				SdbListIter *iter;
+				SdbKv *kv;
+				SdbList *sdbls = sdb_foreach_list (core->anal->sdb_meta, true);
+				ut8 out[len];
+				ut8 c;
+				int i = 0;
+
+				for (int j = 0; j < len; j ++) {
+					ls_foreach (sdbls, iter, kv) {
+						char hex_str[2];
+						memcpy(hex_str, sdbkv_key (kv)+2, sizeof(char)*2);
+						int hex = (int)strtol(hex_str, NULL, 16);
+
```

Comment:
Thats an overkill to load a byte array. We can define those charsets in .h and set a pointer to the array

Reply:
@radare do you just want I move char hex_str[2]; on a .h? Do you expect something more?

Reply:
just fn translate(char ch) {return charset_table[ch]}